src/textual/widgets/_progress_bar.py

```diff
@@ -72,27 +96,47 @@ def __init__(
         self._start_time = None
         self._percentage = None
 
+    def validate_colorscheme(self, colorscheme: str) -> str:
+        if colorscheme is None:
+            colorscheme = "default"
+        if colorscheme not in _VALID_BAR_COLORSCHEMES:
+            raise InvalidBarColorScheme(
+                f"Valid bar color shemes are {friendly_list(_VALID_BAR_COLORSCHEMES)}"
+            )
+        return colorscheme
+
+    def validate_thickness(self, thickness: int) -> int:
```

Comment:
Fixed.

Reply:
When I said "it can be used here", I also meant in the typing sense; something like

```suggestion
    def validate_thickness(self, thickness: int) -> BarThickness:
```